bin/conjur-cli.rb

```diff
@@ -197,13 +197,13 @@ def test_select
   cgrp.desc "Retrieve a role's API key"
   cgrp.command :"retrieve-key" do |c|
     c.action do |global_options,options,args|
-      exit_now! "Must specify role ID" unless args.count == 1
-      
-      role_id = args.shift
-      
+      help_now! "specify at least one role id" unless args.count > 0
```

Comment:
You can also stick 
```ruby
arguments :strict
subcommand_option_handling :normal
```
in the `App` class, declare arguments with `arg` instead of `arg_name` and have GLI check the arguments for you, eg:
```ruby
arg :role_id, :multiple
c.action { ... }
```

This will also automatically make it bail if arguments are given but not declared, so most of all these checks that are sprinkled around can come off.